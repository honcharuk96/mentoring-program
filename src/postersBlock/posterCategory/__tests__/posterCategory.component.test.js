import React from 'react';
import PosterCategoryComponent from '../posterCategory.component';
import { render } from '@testing-library/react';

describe('<PosterCategoryComponent />', () => {
  it('should render PosterCategoryComponent', () => {
    const { asFragment } = render(<PosterCategoryComponent />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('should render PosterCategoryComponent with posters', () => {
    const genres = ['Drama', 'Romance'];
    const { getByText } = render(<PosterCategoryComponent genres={genres} />);
    const el = getByText('Romance');
    expect(el);
  });
});
