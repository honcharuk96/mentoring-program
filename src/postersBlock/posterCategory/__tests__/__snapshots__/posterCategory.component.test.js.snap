// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PosterCategoryComponent /> should render PosterCategoryComponent 1`] = `
<DocumentFragment>
  <div
    class="posterCategorystyled__PosterCategory-sc-szupof-0 gVdyHw"
  />
</DocumentFragment>
`;
